config/content_mappings.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
(As discussed, these keys would have been fine in YAML, but obviously fine to leave now)